src/envoy/prototype/server_config.pb.txt

```diff
@@ -2,5 +2,5 @@ cloud_tracing_config {
   force_disable: true
 }
 mixer_options {
-  mixer_server: "mixer_server"
-}
\ No newline at end of file
+  mixer_server: "localhost:9091"
```

Comment:
Ok, might want to explain whether GRPC does DNS resolution on every request or if it is blocking.